automation-tests/test/utilities.ts

```diff
@@ -0,0 +1,337 @@
+}
+
+function log(message: string) {
+  console.log(message);
+}
+
+async function clickFilePathOkButton(): Promise<void> {
+  const okButton = await $('*:not([style*="display: none"]).quick-input-action .monaco-button');
+  await okButton.click();
+  await utilities.pause(1);
+}
+
+async function openCommandPromptWithCommand(workbench: Workbench, command: string): Promise<InputBox | QuickOpenBox> {
+  const prompt = await workbench.openCommandPrompt();
+  await prompt.wait(5000);
+
+  await prompt.setText(`>${command}`);
+  await pause(1);
+
+  return prompt;
+}
+
+async function executeQuickPick(workbench: Workbench, command: string): Promise<InputBox | QuickOpenBox> {
+  const prompt = await openCommandPromptWithCommand(workbench, command);
+  await selectQuickPickItem(prompt, command);
+
+  return prompt;
+}
+
+async function selectQuickPickItem(prompt: InputBox | QuickOpenBox, text: string): Promise<void> {
+  const quickPicks = await prompt.getQuickPicks();
+  for (const quickPick of quickPicks) {
+      const label = await quickPick.getLabel();
+      if (label === text) {
+          await quickPick.select();
+          return;
+      }
+  }
+
+  throw new Error(`Quick pick item ${text} was not found`);
+}
+
+async function getStatusBarItemWhichIncludes(statusBar: StatusBar, title: string): Promise<WebdriverIO.Element> {
+  const items = await statusBar.item$$;
+  for (const item of items) {
+    const itemTitle = await item.getAttribute(statusBar.locators.itemTitle);
+    if (itemTitle.includes(title)) {
+        return item;
+    }
+  }
+
+  throw new Error(`Status bar item containing ${title} was not found`);
+}
+
+async function waitForNotificationToGoAway(workbench: Workbench, notificationMessage: string, timeout: number): Promise<void> {
+  // Change timeout from seconds to milliseconds
+  timeout *= 1000;
+
+  pause(5);
+  const startDate = new Date();
+  while (true) {
+    let notificationWasFound = await notificationIsPresent(workbench, notificationMessage);
+    if (!notificationWasFound) {
+      return;
+    }
+
+    const currentDate = new Date();
+    const secondsPassed = Math.abs(currentDate.getTime() - startDate.getTime()) / 1000;
+    if (secondsPassed >= timeout) {
+      throw new Error(`Exceeded time limit - notification "${notificationMessage}" is still present`);
+    }
+  }
+}
+
+async function notificationIsPresent(workbench: Workbench, notificationMessage: string): Promise<boolean> {
+  const notifications = await workbench.getNotifications();
+  for (const notification of notifications) {
+    const message = await notification.getMessage();
+    if (message === notificationMessage) {
+      return true;
+    }
+  }
+
+  return false;
+}
+
+async function attemptToFindNotification(workbench: Workbench, notificationMessage: string, attempts: number): Promise<boolean> {
+  while (attempts > 0) {
+    if (notificationIsPresent(workbench, notificationMessage)) {
```

Comment:
this needs an await b/c `notificationIsPresent` is async

Reply:
Good find, I'll push a fix